thelper/tasks/utils.py

```diff
@@ -4,19 +4,17 @@
 and the base interface used to define new tasks.
 """
 
-import collections
 import logging
 import re
-from typing import Union
+import typing
 
 import thelper.typedefs
 import thelper.utils
 
 logger = logging.getLogger(__name__)
 
 
-def create_task(config):
-    # type: (Union[Task, thelper.typedefs.ConfigDict, str]) -> Task
+def create_task(config: typing.Union[thelper.typedefs.ConfigDict, typing.AnyStr]) -> "Task":
```

Comment:
It doesn't need `thelper.task.Task` ?

Reply:
since it's defined in the same file, it works without the full spec (magicmagicmagic)

Reply:
(now in quotes because it's defined below the 1st use of the type)

Reply:
👍